packages/backup-restore/src/config/configstore.ts

```diff
@@ -0,0 +1,27 @@
+import fs from "fs";
+import Configstore from "configstore";
+
+const pkg = JSON.parse(fs.readFileSync("./package.json", "utf8"));
```

Comment:
This means that, when the script is invoked, it should look up a file called `package.json` in the current directory.
This is not what we want.